internal/ingest/preprocessing_task.go

```diff
@@ -0,0 +1,24 @@
+package ingest
+
+import (
+	"time"
+
+	"github.com/artefactual-sdps/enduro/internal/enums"
+)
+
+type Task struct {
+	// Name is the name of the task.
+	Name string
+
+	// Message is a human readable description of the task operations or result.
+	Message string
+
+	// Outcome indicates the completion state of the task.
+	Outcome enums.IngestTaskOutcome
+
+	// StartedAt is the timestamp of the task initiation.
+	StartedAt time.Time
+
+	// StartedAt is the timestamp of the task completion.
+	CompletedAt time.Time
+}
```

Comment:
I think it's better to have this in the preprocessing package,

Reply:
This was my poorly thought out, last minute attempt to shorten "preprocessing" (14 chars) to "ingest" (7 chars) and shorten awfully long names like "PreprocessingTaskOutcomeValidationFailure". :grimacing:  I went looking at some of Dan's newer architecture diagrams for SFA, and I thought "Ingest" was the new "official" name for "Preprocessing".  I talked with Dan & Sara today though, and  "Ingest" refers to everything before Preservation, which would include preprocessing and processing. :(

I'll switch back from ingest -> preprocessing.